Makefile

```diff
@@ -1,11 +1,11 @@
 SHELL := /bin/bash
-php_versions ?= 73 74
 layer ?= *
 parallel = $(if $(shell which parallel),true,false)
+resolve_php_versions = $(or $(php_versions),`jq -r '.php | join(" ")' ${1}/config.json`)
```

Comment:
imply to have jq installed, should I change of something else? I can try doing it in php.
should look like that:
```sh
php -r 'echo join(" ", json_decode(file_get_contents("${1}/config.json"), true)["php"]);'
```

Reply:
Yeah, But that imply that you have PHP installed =)

As long as it works on github actions then Im fine either way.

Reply:
Nice with `or` btw.

Reply:
Yes, this is fully BC and very handy when you just want to rebuild some specific layer variant.